cmd/zc_traverser_s3_service.go

```diff
@@ -72,7 +78,18 @@ func (t *s3ServiceTraverser) traverse(processor objectProcessor, filters []objec
 			err = bucketTraverser.traverse(middlemanProcessor, filters)
 
 			if err != nil {
-				return err
+				if strings.Contains(err.Error(), "301 response missing Location header") {
+					LogStdoutAndJobLog(fmt.Sprintf("skip enumerating the bucket %q , as it's not in the region specified by source URL", v.Name))
+					continue
+				}
+
+				if strings.Contains(err.Error(), "cannot list objects, The specified bucket does not exist") {
+					LogStdoutAndJobLog(fmt.Sprintf("skip enumerating the bucket %q, as it does not exist (perhaps you have a ghost bucket?)", v.Name))
```

Comment:
Is "ghost bucket" a recognised AWS term?  If not, I don't think we can use that term.

Reply:
Trimmed off the portion of the string in parentheses.